src/gui/windows/controls.zig

```diff
@@ -80,4 +126,4 @@ pub fn render() void {
 		onOpen();
 		window.rootComponent.?.verticalList.scrollBar.currentState = oldScroll;
 	}
-}
\ No newline at end of file
+}
```

Comment:
![Screenshot at 2024-09-30 19-05-48](https://github.com/user-attachments/assets/22c36843-9131-47aa-9ac9-eaf17c980b8b)
At default scale the window is too high for the default screen (it's cut off at the bottom). Please make the window smaller to ensure that it works for all resolutions.